docs/src/pages/components/snackbars/CustomizedSnackbars.tsx

```diff
@@ -40,19 +27,19 @@ export default function CustomizedSnackbars() {
   };
 
   return (
-    <div className={classes.root}>
+    <Box sx={{ width: '100%', '& > * + *': { mt: 2 } }}>
```

Comment:
What do you think of using this opportunity to use the Stack component each time possible. For instance, here it seems to simplify the demo, https://next.material-ui.com/components/snackbars/#customized-snackbars